### PR TITLE
chore(website): Update docs and front page

### PR DESCRIPTION
**Pull request by yevgenypats**
This updates the messaging as we expanded our source and destination to fit well for all ELT use-cases.

### CODE REVIEW

**Review by erezrokah — COMMENTED** [review]
We might want to search the entire repo and update readme files, [Homebrew descriptions](https://github.com/cloudquery/cloudquery/blob/795e40d05ef7cb8f5dbe7b5fa94cb2c02e0f1c16/cli/.goreleaser.yaml#L35) and maybe even this repo description on GitHub

File: website/components/pages/home.tsx [hunk 1]
```diff
@@ -126,10 +128,10 @@ export default function Home() {
       <div className="relative from-gray-50 to-gray-100">
         <div className="px-4 py-16 mx-auto sm:pt-20 sm:pb-24 lg:max-w-7xl lg:pt-24">
           <h2 className="nx-text-4xl font-extrabold tracking-tight lg:nx-text-5xl xl:nx-text-6xl lg:text-center dark:text-white">
-            Data analysis, security, auditing, and compliance
+              Built for any team
           </h2>
           <p className="mx-auto mt-4 text-lg font-medium text-gray-400 lg:max-w-3xl lg:text-xl lg:text-center">
-            Leverage SQL to get visibility into your cloud infrastructure and SaaS applications.
+            Engineering, Infrastructure, Security, Marketing, Sales, Support, FinOps
```
Comment (jsonpr) [inline website/components/pages/home.tsx:134]: Thoughts on adding `Platform`?

File: website/content/features.ts [hunk 2]
```diff
@@ -45,14 +33,26 @@ const FEATURES: Features = [
...
+    page: "all",
+  },
+  {
+    name: "Pre-built queries",
+    description: `CloudQuery maintains number of out-of-the-box security and compliance policies for cloud infrastructure.`,
+    Icon: KeyIcon,
+    page: "all",
+  },
+  {
+    name: "Elimnate data silos",
```
Comment (jsonpr) [inline website/content/features.ts:48]:
```suggestion
    name: "Eliminate data silos",
```

File: website/content/features.ts [hunk 3]
```diff
@@ -45,14 +33,26 @@ const FEATURES: Features = [
...
+    description: `Elimnate data silos across your organization, security, infrastructure, marketing and finance teams.`,
```
Comment (jsonpr) [inline website/content/features.ts:49]:
```suggestion
    description: `Eliminate data silos across your organization, security, infrastructure, marketing and finance teams.`,
```

File: website/content/features.ts [hunk 4]
```diff
@@ -45,14 +33,26 @@ const FEATURES: Features = [
...
     Icon: DatabaseIcon,
     page: "all",
   },
   {
-    name: "Raw access to data",
-    description: `Decouple data ingestion and get raw access to your data in structured and unstructured formats.`,
+    name: "Unlimited scale",
+    description: `CloudQuery plugins are statless and can scaled easily horizontally on any executor such as k8s, batch jobs.`,
```
Comment (jsonpr) [inline website/content/features.ts:55]:
```suggestion
    description: `CloudQuery plugins are stateless and can scaled easily horizontally on any executor such as k8s, batch jobs.`,
```

File: website/pages/docs/core-concepts/policies.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 # Policies
 
-CloudQuery uses standard SQL to create security, compliance and cost policies that you can visualize and monitor with your current BI stack such as [Grafana](https://grafana.com/), [Apache Superset](https://superset.apache.org/), [QuickSight](https://aws.amazon.com/quicksight/), [PowerBI](https://powerbi.microsoft.com/en-au/), etc…
+CloudQuery maintains a few official SQL policies, and dashboards for security, compliance and cost that you can visualize and monitor with your current BI stack such as [Grafana](https://grafana.com/), [Apache Superset](https://superset.apache.org/), [QuickSight](https://aws.amazon.com/quicksight/), [PowerBI](https://powerbi.microsoft.com/en-au/), etc…
 
-We provide out-of-the-box SQL policies and dashboards for asset inventories and CSPMs (Cloud Security Posture Management) with popular frameworks such as CIS, HIPPA, PCI-DSS and more.
+Policies, views and dashboard include, asset inventories and CSPMs (Cloud Security Posture Management) with popular frameworks such as CIS, HIPPA, PCI-DSS and more.
```
Comment (jsonpr) [inline website/pages/docs/core-concepts/policies.md:5]:
```suggestion
Policies, views, and dashboard include asset inventories and CSPMs (Cloud Security Posture Management) with popular frameworks such as CIS, HIPPA, PCI-DSS and more.
```

File: website/pages/docs/cq-vs-others/fivetran.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# CloudQuery Vs Fivetran
+
+Fivetran is a closed source managed ELT SaaS solution with a few key differences.
+
+**Key Differences:**
+
+- **Open Source and extendable**: CloudQuery is open-source and has a [pluggable architecture](https://hub.cloudquery.io) which means you can contribute missing resources to existing plugins or you can create your own plugins (Using CloudQuery SDK) in order to grab data from proprietary APIs or other SaaS applications.
+- **High Performance** - CloudQuery SDK and connectors are written in Go utilizing excellent support of go-routines which enables high-performance and low memory usage.
+- **Type System** - CloudQuery SDK supports a rich type system for connectors which enables richer schemas and more accurate data.
+- **Self Hosted** - You can run CloudQuery on your own infrastructure, ensuring that your data doesn't leave your infrastructure. Fivetran only offer a managed solution at the moment.
```
Comment (hermanschaaf) [inline website/pages/docs/cq-vs-others/fivetran.md:10]:
```suggestion
- **Self-hosted** - You can run CloudQuery on your own infrastructure, ensuring that your data doesn't leave your infrastructure. Fivetran only offers a managed solution at the moment.
```

File: website/pages/docs/cq-vs-others/fivetran.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+# CloudQuery Vs Fivetran
+
+Fivetran is a closed source managed ELT SaaS solution with a few key differences.
+
+**Key Differences:**
+
+- **Open Source and extendable**: CloudQuery is open-source and has a [pluggable architecture](https://hub.cloudquery.io) which means you can contribute missing resources to existing plugins or you can create your own plugins (Using CloudQuery SDK) in order to grab data from proprietary APIs or other SaaS applications.
```
Comment (hermanschaaf) [inline website/pages/docs/cq-vs-others/fivetran.md:7]:
```suggestion
- **Open Source and extendable**: CloudQuery is open-source and has a [pluggable architecture](/docs/developers/architecture) which means you can contribute missing resources to existing plugins or you can easily create your own plugins (Using CloudQuery SDK) to grab data from in-house data stores or other SaaS applications.
```

File: website/pages/docs/index.mdx [hunk 8]
```diff
@@ -9,17 +9,17 @@ import Features from "../../components/Features";
 <Features page="docs" detailed={false} />
 
 CloudQuery is an [open-source](https://github.com/cloudquery/cloudquery),
-high-performance data integration platform for security and infrastructure teams.
+high-performance data integration built for developers.
```
Comment (hermanschaaf) [inline website/pages/docs/index.mdx:12]:
```suggestion
high-performance data integration framework built for developers.
```

Comment (hermanschaaf) [reply website/pages/docs/index.mdx:12]: Or platform? Either way, seems like there's a word missing here

File: website/pages/docs/index.mdx [hunk 9]
```diff
@@ -9,17 +9,17 @@ import Features from "../../components/Features";
...
 ## Why CloudQuery?
 
-- **Open source**: Easily extendable plugin architecture. Contribute to our official plugins or develop your own with the CloudQuery SDK.
-- **Blazing fast**: CloudQuery's concurrency system leverages the excellent Go concurrency model with light-weight Goroutines.
-- **Database agnostic**: CloudQuery can store your configuration in any supported destination such as a database, data lake, or streaming platform for further analysis.
-- **Pre-built queries**: CloudQuery maintains a rich set of SQL queries and Grafana dashboards for asset inventory, CSPMs, Security & Compliance, and Cost use-cases.
-- **Raw access to data**: Decouple data ingestion and have raw access to your data, with which you can build your own security stack and re-use
-  best-of-breed tools for querying (SQL,…), transformation (dbt,…), and visualization (Grafana, Preset, Metabase, PowerBI, …).
+- **Open source**: Easily extendable plugin architecture. Contribute to our official plugins or develop your own with CloudQuery SDK.
+- **Blazing fast**: CloudQuery concurrency system utilizes the excellent Go concurrency model with light-weight goroutines.
```
Comment (hermanschaaf) [inline website/pages/docs/index.mdx:21]:
```suggestion
- **Blazing fast**: CloudQuery is optimized for performance, utilizing the excellent Go concurrency model with light-weight goroutines.
```

File: website/pages/docs/index.mdx [hunk 10]
```diff
@@ -9,17 +9,17 @@ import Features from "../../components/Features";
...
+- **Deploy anywhere**: CloudQuery plugins are single binary executables and can be deployed and run anywhere.
+- **Pre-built queries**: CloudQuery maintains number of out-of-the-box security and compliance policies for cloud infrastructure.
```
Comment (hermanschaaf) [inline website/pages/docs/index.mdx:23]:
```suggestion
- **Pre-built queries**: CloudQuery maintains out-of-the-box security and compliance policies for cloud infrastructure.
```

File: website/pages/docs/index.mdx [hunk 11]
```diff
@@ -9,17 +9,17 @@ import Features from "../../components/Features";
...
+- **Eliminate data silos**: Eliminate data silos across your organization, security, infrastructure, marketing and finance teams.
```
Comment (hermanschaaf) [inline website/pages/docs/index.mdx:24]:
```suggestion
- **Eliminate data silos**: Eliminate data silos across your organization, unifying data between security, infrastructure, marketing and finance teams.
```

File: website/pages/docs/index.mdx [hunk 12]
```diff
@@ -9,17 +9,17 @@ import Features from "../../components/Features";
...
+- **Unlimited scale**: CloudQuery plugins are stateless and can scaled easily horizontally on any executor such as k8s, batch jobs.
```
Comment (hermanschaaf) [inline website/pages/docs/index.mdx:25]:
```suggestion
- **Unlimited scale**: CloudQuery plugins are stateless and can be scaled horizontally on any platform, such as VMs, Kubernetes or batch jobs.
```

File: website/pages/index.mdx [hunk 13]
```diff
@@ -1,6 +1,6 @@
 ---
-title: CloudQuery | An open source high performance data integration platform designed for security and infrastructure teams
-description: Sync cloud assets to any database, transform and visualize
+title: CloudQuery | An open source high performance data integration platform for developers
+description: Sync any source assets to any database, transform and visualize
```
Comment (hermanschaaf) [inline website/pages/index.mdx:3]:
```suggestion
description: Sync any source to any database, transform and visualize
```

File: website/pages/docs/cq-vs-others/airbyte.md [hunk 14]
```diff
@@ -0,0 +1,11 @@
+# CloudQuery Vs Airbyte
+
+Both CloudQuery and Airbyte are open source ELT frameworks with a few key differences.
+
+**Key Differences:**
+
+- **High Performance** - CloudQuery SDK and connectors are written in Go utilizing excellent support of go-routines which enables high-performance and low memory usage.
```
Comment (hermanschaaf) [inline website/pages/docs/cq-vs-others/airbyte.md:7]:
```suggestion
- **High Performance** - CloudQuery SDK and connectors are optimized for performance, utilizing Go's support for goroutines to enable high performance and low memory usage.
```

File: website/pages/docs/cq-vs-others/airbyte.md [hunk 15]
```diff
@@ -0,0 +1,11 @@
...
+- **Type System** - CloudQuery SDK supports a rich type system for connectors which enables richer schemas and more accurate data.
+- **Stateless** - CloudQuery architecture is execution agnostic, and can be deployed in any environment such as Kubernetes, ECS, Google Cloud Run, Batch Jobs, etc. Airbyte needs an Airbyte database, backend, workflows engine and other components that need to run and maintained.
```
Comment (hermanschaaf) [inline website/pages/docs/cq-vs-others/airbyte.md:9]:
```suggestion
- **Stateless** - CloudQuery architecture is execution agnostic, and can be deployed in any environment such as Kubernetes, ECS, Google Cloud Run, Batch Jobs, etc. Airbyte needs an Airbyte database, backend, workflows engine and other components that need to be run and maintained.
```

File: website/pages/docs/cq-vs-others/airbyte.md [hunk 16]
```diff
@@ -0,0 +1,11 @@
...
+- **Native as code approach** - CloudQuery was designed from the ground up to be configurable and run as code while in Airbyte it came later as a way to configure Airbyte UI.
```
Comment (hermanschaaf) [inline website/pages/docs/cq-vs-others/airbyte.md:10]:
```suggestion
- **Pipeline as code** - CloudQuery was designed from the ground up to be configurable and run as code, while in Airbyte it was added later as a way to configure the Airbyte UI.
```

File: website/content/features.ts [hunk 17]
```diff
@@ -45,14 +33,26 @@ const FEATURES: Features = [
...
-    name: "Database agnostic",
-    description: `CloudQuery can store your configuration in any supported destination such as database, data lake or streaming platform for further analysis.`,
+    name: "Deploy anywhere",
+    description: `CloudQuery plugins are single binary executables and can be deployed and run anywhere.`,
+    Icon: ChartBarIcon,
+    page: "all",
+  },
+  {
+    name: "Pre-built queries",
+    description: `CloudQuery maintains number of out-of-the-box security and compliance policies for cloud infrastructure.`,
```
Comment (jsonpr) [inline website/content/features.ts:43]:
```suggestion
    description: `CloudQuery maintains a number of out-of-the-box security and compliance policies for cloud infrastructure.`,
```

File: website/pages/docs/cq-vs-others/airbyte.md [hunk 18]
```diff
@@ -0,0 +1,11 @@
...
+- **Connectors** - Airbyte has more source connectors then CloudQuery at the moment. CloudQuery team is maintaining a quality set of official widely used connectors while mostly developing CloudQuery SDK to enable developers to write their own high performance connectors.
```
Comment (hermanschaaf) [inline website/pages/docs/cq-vs-others/airbyte.md:11]:
```suggestion
- **Connectors** - Airbyte has more source connectors than CloudQuery at the moment. The CloudQuery team adds and maintains official, widely-used connectors, while also developing the CloudQuery SDK to enable developers to write their own high-performance connectors.
```

**Review by hermanschaaf — COMMENTED** [review]
Posting a few suggestions before the underlying text goes out of date :)

File: website/components/pages/home.tsx [hunk 19]
```diff
@@ -150,7 +152,7 @@ export default function Home() {
             <div className="space-y-4 sm:space-y-0 sm:mx-auto ">
               <Link href="/docs">
                 <a className="flex items-center justify-center w-full px-8 py-3 text-base font-medium text-white no-underline bg-black border border-transparent rounded-md dark:bg-white dark:text-black betterhover:dark:hover:bg-gray-300 betterhover:hover:bg-gray-700 md:py-3 md:text-lg md:px-10 md:leading-6">
-                  Start Building →
+                  Start Syncing →
```
Comment (hermanschaaf) [inline website/components/pages/home.tsx:155]: 🚀

File: website/content/features.ts (hunk 17 above)
Comment (hermanschaaf) [inline website/content/features.ts:43]:
```suggestion
    description: `CloudQuery maintains a number of out-of-the-box security and compliance policies for cloud infrastructure.`,
```

File: website/content/features.ts [hunk 20]
```diff
@@ -45,14 +33,26 @@ const FEATURES: Features = [
...
+    Icon: KeyIcon,
+    page: "all",
+  },
+  {
+    name: "Eliminate data silos",
+    description: `Eliminate data silos across your organization, security, infrastructure, marketing and finance teams.`,
```
Comment (hermanschaaf) [inline website/content/features.ts:49]:
```suggestion
    description: `Eliminate data silos across your organization, unifying data between security, infrastructure, marketing and finance teams.`,
```

File: website/pages/docs/index.mdx (hunk 8 above)
Comment (yevgenypats) [reply website/pages/docs/index.mdx:12]: yeah platform seems right.

File: website/components/pages/home.tsx [hunk 21]
```diff
@@ -44,25 +44,27 @@ export default function Home() {
...
       <div className="flex flex-col md:flex-row justify-between px-4 pt-16 pb-8 mx-auto sm:pt-24 lg:px-8 w-auto lg:max-w-7xl">
         <div className="flex flex-col justify-between md:mr-4">
           <div>
             <h1 className="max-w-5xl mx-auto nx-text-6xl font-extrabold tracking-tighter leading-[1.1] sm:text-7xl lg:nx-text-8xl xl:nx-text-8xl">
-              Data integration
-              <br className="hidden lg:block" />
-              <span className="pr-1 inline-block text-transparent bg-clip-text bg-gradient-to-r from-green-500 to-blue-500 ">
-                at scale.
-              </span>
+               A frustratingly
```
Comment (hermanschaaf) [inline website/components/pages/home.tsx:55]:
```suggestion
               Frustratingly
```

File: website/content/features.ts [hunk 22]
```diff
@@ -20,18 +20,6 @@ export type Feature = {
...
-  },
-  {
-    name: "CSPM",
-    description: `Customize pre-built open source SQL policies and visualize them with your any of your favorite BI tools.`,
-    Icon: KeyIcon,
-    page: "all",
-  },
   {
     name: "Open source",
     description: `Easily extensible plugin architecture. Contribute to our official plugins or develop your own with CloudQuery SDK.`,
```
Comment (hermanschaaf) [inline website/content/features.ts:25]:
```suggestion
    description: `Extensible plugin architecture. Contribute to our official plugins or develop your own with CloudQuery SDK.`,
```

I know this wasn't changed in the PR, but upon seeing it I realized that "easily extensible" doesn't seem right.

File: website/pages/docs/index.mdx [hunk 23]
```diff
@@ -9,17 +9,17 @@ import Features from "../../components/Features";
...
-- **Blazing fast**: CloudQuery's concurrency system leverages the excellent Go concurrency model with light-weight Goroutines.
-- **Database agnostic**: CloudQuery can store your configuration in any supported destination such as a database, data lake, or streaming platform for further analysis.
-- **Pre-built queries**: CloudQuery maintains a rich set of SQL queries and Grafana dashboards for asset inventory, CSPMs, Security & Compliance, and Cost use-cases.
-- **Raw access to data**: Decouple data ingestion and have raw access to your data, with which you can build your own security stack and re-use
-  best-of-breed tools for querying (SQL,…), transformation (dbt,…), and visualization (Grafana, Preset, Metabase, PowerBI, …).
+- **Open source**: Easily extendable plugin architecture. Contribute to our official plugins or develop your own with CloudQuery SDK.
+- **Blazing fast**: CloudQuery is optimized for performance, utilizing the excellent Go concurrency model with light-weight goroutines.
+- **Deploy anywhere**: CloudQuery plugins are single binary executables and can be deployed and run anywhere.
```
Comment (hermanschaaf) [inline website/pages/docs/index.mdx:22]:
```suggestion
- **Deploy anywhere**: CloudQuery plugins are single-binary executables and can be deployed and run anywhere.
```

File: website/pages/docs/index.mdx [hunk 24]
```diff
@@ -9,17 +9,17 @@ import Features from "../../components/Features";
...
 
 ## Why CloudQuery?
 
-- **Open source**: Easily extendable plugin architecture. Contribute to our official plugins or develop your own with the CloudQuery SDK.
-- **Blazing fast**: CloudQuery's concurrency system leverages the excellent Go concurrency model with light-weight Goroutines.
-- **Database agnostic**: CloudQuery can store your configuration in any supported destination such as a database, data lake, or streaming platform for further analysis.
-- **Pre-built queries**: CloudQuery maintains a rich set of SQL queries and Grafana dashboards for asset inventory, CSPMs, Security & Compliance, and Cost use-cases.
-- **Raw access to data**: Decouple data ingestion and have raw access to your data, with which you can build your own security stack and re-use
-  best-of-breed tools for querying (SQL,…), transformation (dbt,…), and visualization (Grafana, Preset, Metabase, PowerBI, …).
+- **Open source**: Easily extendable plugin architecture. Contribute to our official plugins or develop your own with CloudQuery SDK.
```
Comment (hermanschaaf) [inline website/pages/docs/index.mdx:20]:
```suggestion
- **Open source**: Extensible plugin architecture. Contribute to our official plugins or develop your own with CloudQuery SDK.
```

On the home page we used "extensible". Similar meanings, but let's keep it consistent

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**